src/app/components/operations/register/register.component.ts

```diff
@@ -0,0 +1,99 @@
+import { tokenReference } from '@angular/compiler';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { LoginUser } from 'src/app/models/login-user';
+import { NewUser } from 'src/app/models/new-user';
+import { AuthService } from 'src/app/services/auth.service';
+import { TokenService } from 'src/app/services/token.service';
+
+@Component({
+  selector: 'app-register',
+  templateUrl: './register.component.html',
+  styleUrls: ['./register.component.css']
+})
+
+export class RegisterComponent implements OnInit {
+
+  isLogged = false;
+  newUser: NewUser;
+  suscription: Observable<any>;
+
+  messageError: string;
+  isLoginFail = false;
+  formRegister: FormGroup;
+
+  constructor(private authService: AuthService,
+              private tokenService: TokenService,
+              private router: Router,
+              private formBuilder: FormBuilder) {
+
+                this.formRegister = formBuilder.group({
+                  username: ['', Validators.required],
+                  password: ['', Validators.required],
+                  firstName: ['', Validators.required],
+                  lastName: ['', Validators.required],
+                  email: ['', Validators.required]
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
                  email: ["", Validators.required]
```